app/views/user_mailer/welcome.html.erb

```diff
@@ -1,27 +1,26 @@
+  Welcome to Operation Code!<br />
 </p>
 
+<p>
+  Be on the lookout for an email invite to the <a href="https://operation-code.slack.com">Operation Code Slack community</a>. Slack is where we communicate online in real-time to ask questions, collaborate, and share information. It's a pivotal aspect of our community! If you've never used Slack before, consider watching this <a href="https://www.youtube.com/watch?v=9RJZMSsH7-g">instructional video</a> on using the application. <br />
+</p>
 
 <p>
-  We are a decentralized organization, and we work both <a href="https://community.operationcode.org/">online</a> and in <a href="https://www.meetup.com/pro/operationcode/">local chapters</a> across the country. Please introduce yourself on our <a href="https://community.operationcode.org/c/introductions">forums</a>, and come join one of our <a href="https://www.meetup.com/pro/operationcode/">meetups in your local area!</a> <br />
-  Also, be sure to check your email for an invite to the Operation Code Slack community (<a href="https://operation-code.slack.com">https://operation-code.slack.com</a>).<br />
-  Slack is where we communicate online in real-time to ask questions, collaborate, and share information. Once in Slack (you'll probably want to download the app), introduce yourself in the #general channel. An example could be: <br />
+  You've just joined a community of 3000+ software developers, veterans, and military spouses, who are all looking to further their skills and careers in tech! We are a decentralized organization, and we work both online (via Slack) and in <a href="https://www.meetup.com/pro/operationcode/">local chapters across the country.</a> We highly recommend giving yourself an introduction in the #general channel on Slack, and to sign up to go to your next local meetup! An example could be: <br />
   "Hi all. My name is David. I'm from Washington and I'm interested in web development!"
   Please read our <a href="http://op.co.de/code-of-conduct">Code of Conduct</a>, as well.
 </p>
 
 <p>
-  The Operation Code website uses React, a JavaScript framework, and Ruby on Rails. If you want to dive in and help, you can get started by reading our <a href="https://github.com/OperationCode/operationcode/blob/master/CONTRIBUTING.md">docs</a>, or asking around in the #oc-projects Slack channel.
-  If you're interested in our mentorship program, fill out our application form as either a <a href="http://op.co.de/mentor-enrollment">mentor</a> or <a href="http://op.co.de/mentor-request">mentee</a>, also!
+  Are you interested in helping Operation Code's website through code or design, and building your experience and portfolio? We're open source and love onboarding new developers! <a href="https://github.com/OperationCode/START_HERE">Read more here</a>, and join the #oc-projects channel on Slack - feel free to ask about anything.
```

Comment:
Let's use 

`<a href="https://operation-code.slack.com/messages/C04FASBU9">#oc-projects</a>`

Reply:
I kind of like having it as join the #oc-projects channel on Slack - as the direct link might be a little funky depending on where/how someone is running Slack.

Reply:
That works with me :)